Makefile

```diff
@@ -712,6 +712,15 @@ test-go-unit:
 		| tee $(TEST_LOG_DIR)/unit.json \
 		| gotestsum --raw-command -- cat
 
+# Runs tbot unit tests
+.PHONY: test-go-unit-tbot
+test-go-unit-tbot: FLAGS ?= -race -shuffle on
+test-go-unit-tbot: SUBJECT ?= $(shell go list ./... | grep -E '(tool/tbot|lib/tbot)')
+test-go-unit-tbot:
+	$(CGOFLAG) go test -cover -json -tags "$(PAM_TAG) $(FIPS_TAG) $(BPF_TAG) $(RDPCLIENT_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)" $(PACKAGES) $(SUBJECT) $(FLAGS) $(ADDFLAGS) \
```

Comment:
The `tbot` tests probably don't care about a lot of these tags, right? I can't imagine you need BPF, Touch ID, RDP client, etc.

Reply:
Agreed - we shouldn't need any of these tags bar FIPS.